Userland/Libraries/LibC/sys/xattr.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2021, the SerenityOS developers.
+ * Copyright (c) 2022, Kenneth Myhra <kennethmyhra@serenityos.org>.
```

Comment:
Adding copyright for stub functions is a bold move, but I'll allow it :^)

Reply:
Agree, to my defense I added all the other stubs in the file too 🫣